docker/docker_client.go

```diff
@@ -561,6 +562,12 @@ func (c *dockerClient) detectPropertiesHelper(ctx context.Context) error {
 		return nil
 	}
 
+	// We overwrite the TLS clients `InsecureSkipVerify` only if explicitly
+	// specified by the system context
+	if c.sys != nil && c.sys.DockerInsecureSkipTLSVerify != types.OptionalBoolUndefined {
+		c.tlsClientConfig.InsecureSkipVerify = c.sys.DockerInsecureSkipTLSVerify == types.OptionalBoolTrue
+	}
```

Comment:
This, + `doHTTP`, works, but `tlsclientconfig.NewTransport()` is a tiny bit expensive; why not create the `httpClient` once at this point, and store it in a field (like the old `client` used to be, but now in the `// The following members are detected registry properties:` section)

Reply:
Fix proposed in https://github.com/containers/image/pull/617 .